winpr/libwinpr/comm/comm_serial_sys.c

```diff
@@ -37,6 +37,12 @@
 #include <winpr/crt.h>
 #include <winpr/wlog.h>
 
+#if defined __MIPSEL__ && defined _LP64
```

Comment:
On Debian freerdp2 package fails for all mips architectures (mips, mipsel and mips64el) due to missing CMSPAR define.
Maybe it would be better to use just __mips__ flag, to cover all mips arches.